scripts/heroku.sh

```diff
@@ -5,6 +5,8 @@ export INTEGRATE_SOCKETIO=false
 # socketio has problems with celery "blocking" tasks
 # also socketio is not used in a celery task so no problem to turn it off
 chmod -R 0777 ./static
+# FIXME: Need to revisit this, why "&" at the end? Because it causes logging hidden,
+# making us difficulty finding bug if something goes wrong.
```

Comment:
This runs on heroku, so we need to run both commands using same script. The logs are not hidden  when using &

Reply:
I remember wrong about behaviour of `&`. Yes. It doesn't hide the logs. Thanks for correcting me.

But running many commands from the same shell script then send to background like this will make the log from them mixed together, which cause trouble for debugging.

But if this is Heroku requirements, ok, let's pass on it.